tests/checker_integration_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+TEST_DATABASE_NAME=import_test
+CHECKER_EXEC="../bin/checker --host ${MYSQL_HOST} --port ${MYSQL_PORT} --user root ${TEST_DATABASE_NAME}"
+MYSQL_EXEC="mysql -h ${MYSQL_HOST} -P ${MYSQL_PORT} -u root"
+
+init(){
+    ${MYSQL_EXEC} -e "drop database if exists ${TEST_DATABASE_NAME};"
+    checkExit
```

Comment:
Please use `set -e` instead of having `checkExit` everywhere.

```sh
set -e

init() {
    ${MYSQL_EXEC} -e "drop database if exists ${TEST_DATABASE_NAME};"
    ${MYSQL_EXEC} -e "create database ${TEST_DATABASE_NAME};"
    ${MYSQL_EXEC} -e "create table ${TEST_DATABASE_NAME}.ta(a int primary key, b double, c varchar(10), d date unique, e time unique, f timestamp unique, g date unique, h datetime unique, i year unique);"
    ...
}
```